src/metabase/lib/temporal_bucket/util.cljc

```diff
@@ -0,0 +1,55 @@
+(ns metabase.lib.temporal-bucket.util
+  (:require
+   [metabase.util.i18n :as i18n])
+  #?(:cljs (:require-macros [metabase.lib.temporal-bucket.util])))
+
+(defmacro temporal-interval-tru
```

Comment:
It's not possible to define this macro in `metabase.lib.temporal-bucket` due to CLJS compilation issues.